src/components/FutureVisits/FutureVisits.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import TripCard from '../TripCard/TripCard';
+
+const FutureVisits = ({futureTrips}) => {
+
+  const userTrips = futureTrips.map(trip => {
+      return(
+        <TripCard 
+          image={trip.image}
```

Comment:
You're accessing trip dot quite a few times, does it make sense to destructure?

Reply:
Good call - I don't know why I didn't think to destructure inside the map!